pkgs/top-level/all-packages.nix

```diff
@@ -5767,6 +5767,10 @@ let
 
   geoclue2 = callPackage ../development/libraries/geoclue/2.0.nix {};
 
+  geoip-full = makeOverridable (callPackage ../development/libraries/geoip) {
```

Comment:
Here too. Something like `geoipWithGeoliteDatabase` would be more conventional (and specific).